src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/common/util/RandomDataGenerators.kt

```diff
@@ -4,28 +4,21 @@ private val upperCase = ('A'..'Z').toList()
 private val lowerCase = ('a'..'z').toList()
 private val digits = ('0'..'9').toList()
 
-fun randomAlphanumericString(length: Int) = (upperCase + lowerCase + digits)(length).asString()
-
-fun randomUppercaseString(length: Int) = upperCase(length).asString()
+fun Sequence<Char>.asString() = fold(StringBuilder(), StringBuilder::append).toString()
+private operator fun Collection<Char>.invoke(n: Int) = generateSequence { random() }.take(n)
 
-fun randomLowercaseString(length: Int) = lowerCase(length).asString()
+private fun space() = sequenceOf(' ')
+fun word(length: IntRange) = lowerCase(length.random())
+fun capitalisedWord(length: IntRange) = upperCase(1) + lowerCase((length).random() - 1)
 
+fun randomAlphanumericString(length: Int = 6) = (upperCase + lowerCase + digits)(length).asString()
```

Comment:
Just being picky but some functions are prefixed with random but the class is called RandomDataGenerator so maybe drop random from each function? (

Reply:
Done 👍